extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.configuration.bindings.js

```diff
@@ -0,0 +1,87 @@
+angular.module('PaperUI.controllers.configuration.bindings', [ 'ngRoute' ]).config([ '$routeProvider', function($routeProvider) {
+    // Configure the routes for this controller
+    $routeProvider.when('/configuration/bindings', {
```

Comment:
Ok, you will make this difficult I think to add routes. On other projects I've worked on including the angular UI project, the routea are with the implementation controller. 
If you want all routes defined in the init block of the app controller then it will not be possible to add fragments with new conteollers since this block can't be easily changed.